### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
First Commit

### CODE REVIEW

File: fsr-change-management/connector.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class ChnageManagement(Connector):
+    def execute(self, config, operation, params, *args, **kwargs):
+        logger.info('In execute() Operation: {}'.format(operation))
+        try:
+            operation = operations.get(operation)
+            return operation(config, params, *args, **kwargs)
+        except Exception as err:
+            logger.error('{}'.format(err))
```
Comment (cs-pranjali-munfan) [inline fsr-change-management/connector.py]: use logger.exception

File: fsr-change-management/info.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        "tooltip": "Certificate .json file used to connect to the development or staging server",
+        "description": "Certificate .json file used to connect to the development or staging server"
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "sync_change_request",
+      "title": "Sync Change Request",
+      "description": "",
```
Comment (cs-pranjali-munfan) [inline fsr-change-management/info.json]: add action description

File: fsr-change-management/info.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      "output_schema": {
+        "task_id": ""
+      },
+      "parameters": [
+        {
+          "title": "Change Request Payload",
+          "type": "text",
+          "name": "cr_payload",
+          "description": "Specify the Change Request record payload which you want to sync with development server or staging server",
+          "tooltip": "Specify the Change Request record payload which you want to sync with development server or staging server",
```
Comment (cs-pranjali-munfan) [inline fsr-change-management/info.json:55]: add sample example

File: fsr-change-management/operations.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        public_key = file_data.get('public_key', '')
+        private_key = file_data.get('private_key', '')
+        api_endpoint = file_data.get('api_endpoint', '')
+        full_url = api_endpoint
+        method = 'POST'
+        payload = params.get('cr_payload')
+        data = json.dumps(payload)
+        auth_header = __get_headers(full_url, data, private_key, public_key)
+        headers = {'Authorization': auth_header}
+        response = requests.request(method, full_url, headers=headers, data=data, verify=False)
```
Comment (cs-pranjali-munfan) [inline fsr-change-management/operations.py]: take verify SSL from configuration

File: fsr-change-management/info.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply fsr-change-management/info.json:55]: samples are added in sample playbooks

**Review by cs-pranjali-munfan — APPROVED** [review]

**Pull request merged**